src/orders/components/OrderFulfillLine/OrderFulfillLine.tsx

```diff
@@ -86,14 +83,10 @@ export const OrderFulfillLine: React.FC<OrderFulfillLineProps> = props => {
         className={classes.colName}
         thumbnail={line?.thumbnail?.url}
         badge={
-          isPreorder || !line?.variant ? (
+          !line?.variant ? (
```

Comment:
This component (OrderFulfillLine.tsx) seems like a big one - if you touch this - how about splitting some parts into smaller units? Think of extracting components as well as functions that do something eg. display some text, prepare some event handlers and so on